### DISCUSSION TITLE
I would like to reverse the color of the figure.

### DISCUSSION THREAD

**Discussion by Python-Zhou** [ImageMagick/ImageMagick#2160] [Help] [answered]
I would like to change the blue color into red and red one into blue. Just full reserve the RGB
color. However, when I use the negate function, it does not meet my requirement. I do not know how can I. Looking forwad to your help.
Origin:
![origin](https://user-images.githubusercontent.com/40736562/84901474-19c83180-b0de-11ea-907e-e2d5886546df.png)
After negate:
![negate](https://user-images.githubusercontent.com/40736562/84901633-57c55580-b0de-11ea-9277-ea650d2c2de3.png)

**Comment by snibgo** [comment, marked as answer]
You want to swap the red and blue channels. Is that right?
```
magick in.png -channel RGB -separate -swap 0,2 -combine out.png
```

**Reply by Python-Zhou** [reply]
Great great thanks！
